deployment_scripts/linode-marketplace-kali-linux/kali-linux-deploy.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -e
+if [ "${DEBUG}" == "NO" ]; then
+  trap "cleanup $? $LINENO" EXIT
+fi
+
+## Kali
+#<UDF name="everything" label="Would you like to Install the Kali Everything Package?" oneOf="Yes,No" default="Yes">
+#<UDF name="headless" label="Would you like to Install the Kali Headless Package?" oneOf="Yes,No" default="No">
+#<UDF name="vnc" label="Would you like to setup VNC to access Kali XFCE Desktop" oneOf="Yes,No" default="Yes">
+#<UDF name="vnc_username" label="The VNC user to be created for the Linode. The username accepts only lowercase letters, numbers, dashes (-) and underscores (_)">
+#<UDF name="vnc_password" label="The password for the limited VNC user">
+
+## Linode/SSH Security Settings
+#<UDF name="user_name" label="The limited sudo user to be created for the Linode: *All lowercase*">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+
+# git repo
+export GIT_REPO="https://github.com/akamai-compute-marketplace/marketplace-apps.git"
+export WORK_DIR="/tmp/marketplace-apps" 
+export MARKETPLACE_APP="apps/linode-marketplace-kali-linux"
+
+# enable logging
+exec > >(tee /dev/ttyS0 /var/log/stackscript.log) 2>&1
+
+function cleanup {
+  if [ -d "${WORK_DIR}" ]; then
+    rm -rf ${WORK_DIR}
+  fi
+
+}
+
+function udf {
+  local group_vars="${WORK_DIR}/${MARKETPLACE_APP}/group_vars/linode/vars"
+
+  sed 's/  //g' <<EOF > ${group_vars}
+  # Kali Linux settings
+  kali_package: "{{ 'kali-linux-everything' if '${EVERYTHING}' == 'Yes' else 'kali-linux-headless' if '${HEADLESS}' == 'Yes' else '' }}"
+
+  # VNC settings
+  vnc_enabled: "{{ '${VNC}' == 'Yes' }}"
```

Comment:
We are making boolean conditional in the playbook. This is a string. We need to translate 'Yes' or 'No' to either be `True` or `False` in the vars file.